content/blog/setting-up-hierarchical-namespaces-in-kubernetes-in-hpe-greenlake-for-private-cloud-enterprise.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Setting up hierarchical namespaces in Kubernetes in HPE GreenLake for
+  Private Cloud Enterprise
+date: 2024-06-13T08:18:47.600Z
+author: Guoping Jia
+authorimage: /img/guoping.png
+disable: false
+tags:
+  - HPE GreenLake for Private Cloud Enterprise
+  - Kubernetes
+  - Namespaces
+---
+<style> li { font-size: 27px; line-height: 33px; max-width: none; } </style>
+
+This blog post shows the process to set up the hierarchical namespaces in Kubernetes (K8s) in HPE GreenLake for Private Cloud Enterprise. It demonstrates how easy to manage namespace relationships, propagate configurations and resources, and enforce access control policies using the hierarchical namespace in K8s for the cluster administrators. 
+
+
+### Overview
+
+[HPE GreenLake for Private Cloud Enterprise: Containers](https://www.hpe.com/us/en/greenlake/containers.html), one of the HPE GreenLake cloud services available on the HPE GreenLake for Private Cloud Enterprise, allows customers to create a K8s cluster and deploy containerized applications to the cluster. It provides an enterprise-grade container management service using open source K8s.  
+
+### Prerequisites
+
+Before starting, make sure you have the following:

```

Comment:
Can we just say **Prerequisites** ? The first sentence 'before starting...' is not required.